src/api/DomainObjects.ts

```diff
@@ -0,0 +1,115 @@
+    AFRL = "AFRL",
+    AFIMSC = "AFIMSC",
+    AFSC = "AFSC",
+    AFNWC = "AFNWC",
+    AFTC = "AFTC"
+}
+
+export enum Stages {
+    BUYER_REVIEW = "Buyer Review",
+    CO_INITIAL_REVIEW = "CO Initial Review",
+    SBP_REVIEW = "SBP Review",
+    SBA_PCR_REVIEW = "SBA PCR Review",
+    CO_FINAL_REVIEW = "CO Final Review",
+    COMPLETED = "Completed"
+}
+
+export enum SetAsideRecommendations {
+    SET_ASIDE = "SB Set-Aside",
+    EIGHT_A = "8(a)",
+    HUBZONE_SB = "HUBZone-SB",
+    SDVOSB = "SDVOSB",
+    EDWOSB = "EDWOSB",
+    WOSB = "WOSB",
+    OTHER = "Other Set-Aside",
+    OTHER_THAN_FULL_OPEN = "Other than Full and Open",
+    FULL_OPEN = "Full and Open"
+}
+
+export enum ProcessTypes {
+    DD2579 = "DD2579",
+    ISP = "ISP"
+}
+
+export interface INote {
+    Id: number,
+    ProcessId: number,
+    Text: string,
+    Author: IPerson,
+    Modified: DateTime
+}
+
+export interface IProcess {
+    Id: number,
+    ProcessType: ProcessTypes,
+    SolicitationNumber: string,
+    ProgramName: string,
+    ParentOrg: ParentOrgs, // I'm not sure this actually needs to be saved, it isn't reported to the user
+    Org: string,
+    Buyer: IPerson,
+    ContractingOfficer: IPerson,
+    SmallBusinessProfessional: IPerson,
+    SboDuration: number,
+    ContractValueDollars: number,
+    SetAsideRecommendation: SetAsideRecommendations, 
+    MultipleAward: boolean,
+    Created: DateTime,
+    Modified: DateTime,
+    CurrentStage: Stages,
+    CurrentAssignee: IPerson, // not sure if we need this since we could probably just extrapolate who is currently assigned based on the status
+    SBAPCR: IPerson,
+    BuyerReviewStartDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    BuyerReviewEndDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    COInitialReviewStartDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    COInitialReviewEndDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    SBPReviewStartDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    SBPReviewEndDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    SBAPCRReviewStartDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    SBAPCRReviewEndDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    COFinalReviewStartDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
+    COFinalReviewEndDate: DateTime, // not sure if needed, it is not ever shown to the user, it may only be used in the reports page
```

Comment:
How do we want to handle checking to see if we ever use these and then getting rid of them if we don't?